packages/react-components/src/components/ImageGallery.tsx

```diff
@@ -159,6 +164,10 @@ export const ImageGallery = (props: ImageGalleryProps): JSX.Element => {
             src={image.imageUrl}
             className={mergeStyles(imageStyle, styles?.image)}
             alt={image.altText || defaultAltText}
+            onError={() => {
+              console.log('Unable to load Image');
```

Comment:
Might not be a bad idea to have an `onError` in the props so we can send an error back up. Just a thought

Reply:
> Might not be a bad idea to have an `onError` in the props so we can send an error back up. Just a thought

I like this idea. Will update the API